### DISCUSSION TITLE
Dokku for self-hosting generic apps?

### DISCUSSION THREAD

**Discussion by MarcCoquand** [dokku/dokku#6267] [Q&A]
Hi!

I'm looking into dokku as a way to manage a VPS with multiple different self-hosted apps on a single instance to make better use of it. Specifically, I'd like to host FreshRSS (has a docker image), Funkwhale (also has a docker image) and also my own personal projects or scripts. Would dokku help me out in this case?

Thank you so much in advance.

**Comment by josegonzalez** [comment]
Yeah that should be fine. We support deploying docker images via `git:from-image`.

See [this doc](https://dokku.com/docs/deployment/methods/git/?h=image#initializing-an-app-repository-from-a-docker-image) for more info on how that works.

**Comment by MarcCoquand** [comment]
Awesome, I just tried it out with the Freshrss docker image and it
worked like magic! Thank you.
